index.html

```diff
@@ -49,6 +49,6 @@ <h3> Market </h3>
 </p>
 <p>
 Insert a list of profitable python code ideas
-Such as packaging seven different stock watching bots and aggregating results into one recommendation. If this shows up the merge was successful.
```

Comment:
Did you mean to have this in all caps? Can you make it the same casing as the rest of the sentence?

Reply:
Ok. I can do that.